src/unicodeMake.js

```diff
@@ -0,0 +1,74 @@
+/* eslint no-console:0 no-undef:0 */
+
+// This is an internal Node tool, not part of the KaTeX distribution,
+// whose purpose is to generate unicodeSymbols.js in this directory.
+// In this way, only this tool, and not the distribution/browser,
+// needs String's normalize function.
+//
+// This tool should be run (via `node unicodeMake.js` or `make unicode`)
+// whenever KaTeX adds support for new accents, and whenever
+// the Unicode spec adds new symbols that should be supported.
+
+console.log("// This file is GENERATED by unicodeMake.js. DO NOT MODIFY.");
+console.log("");
+
+// Define accents locally and output the definition to unicodeSymbols.js
+accentsCode = `accents = {
+    '\\u0301': {text: "\\\\'", math: '\\\\acute'},
+    '\\u0300': {text: '\\\\\`', math: '\\\\grave'},
+    '\\u0308': {text: '\\\\"', math: '\\\\ddot'},
+    '\\u0303': {text: '\\\\~', math: '\\\\tilde'},
+    '\\u0304': {text: '\\\\=', math: '\\\\bar'},
+    '\\u0306': {text: '\\\\u', math: '\\\\breve'},
+    '\\u030c': {text: '\\\\v', math: '\\\\check'},
+    '\\u0302': {text: '\\\\^', math: '\\\\hat'},
+    '\\u0307': {text: '\\\\.', math: '\\\\dot'},
+    '\\u030a': {text: '\\\\r'},
+    '\\u030b': {text: '\\\\H'},
+}`;
+eval(accentsCode);
```

Comment:
I'm confused why the `eval` is necessary, can't we just `console.log(accentsCode);` this as is?

Reply:
A better solution might be to do:
```
const {accents} = require('./unicodeSymbols.js');

const encodedAccents = {};
for (const [key, value] of Object.values(accents)) {
    const encodedEntry = {};
    encodedAccents[encode(key)] = encodedEntry;
    if (value.text) {
        encodedEntry.text = encode(value.text);
    }
    if (value.math) {
        encodedEntry.math = encode(value.math);
    }
}
```
This way we have one source of truths for the accents.